pkg/controllers/management/cis/register.go

```diff
@@ -0,0 +1,31 @@
+package cis
+
+import (
+	"context"
+
+	"github.com/rancher/types/config"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Register initializes the controllers and registers
+func Register(ctx context.Context, management *config.ManagementContext) {
+	logrus.Infof("Registering CIS scheduled scan controller")
```

Comment:
Do we want this at Info?

Reply:
This gets printed only once, should be ok.

Reply:
We don't really need this, the framework will log each controller that registers.

Reply:
Sure, will change to debug.